src/initFolders.ts

```diff
@@ -0,0 +1,35 @@
+import {makeSurePathExists} from './core/utils';
+import path from 'path';
+import config from 'config';
+
+const initFolders = async () => {
+    const datadir: string = config.get('datadir');
```

Comment:
Please use `resolveHome` here. And also please double-check all the other places.